src/App/App.tsx

```diff
@@ -94,6 +95,12 @@ class App extends Component {
                 path={FI_URL}
                 component={ActiveFocusInvestigation}
               />
+              <ConnectedPrivateRoute
```

Comment:
Do you think its possible to just modify the existing route to the `ActiveFocusInvestigation` by making it parametrized as you've done here; such that you would not need to create an entirely new parameterized route.